docs/learning-powershell/README.md

```diff
@@ -100,7 +100,7 @@ Note that all bash commands should continue working on PowerShell session.
 | rm -r &lt;folderName>           |Remove-Item &lt;folderName> -Recurse   |Delete a folder
 | find -name build*               |Get-ChildItem build* -Recurse          |Find a file or folder starting with 'build'
 | grep -Rin "sometext" --include="*.cs" |Get-ChildItem -Recurse -Filter *.cs <br> \| Select-String -Pattern "sometext" | Recursively case-insensitive search for text in files
-
+| curl dominio.com | Invoke-WebRequest dominio.com | Transfer data from or to a web from differentes protocols
```

Comment:
Maybe `Transfer data from web`?

Reply:
I think `Invoke-RestMethod` is more similar to `curl` than `Invoke-WebRequest`.  Also, the example does work with curl and it's better to use a more generic URL:

> curl https://github.com
> Invoke-RestMethod https://github.com

For the description, maybe: Transfer data to or from the web

Reply:
Ok, I'll make the changes.